### PR TITLE
libres-993 Expose file system Forward Model jobs as Workflow jobs

### PR DESCRIPTION
**Pull request by jepebe**
There are several Forward Model jobs that give access to file system operations. Having access to these as Workflow jobs is useful. This PR solves this by moving the common scripts to a higher level and creating Workflow job definitions. 

This solves #993

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by pinkwah** [comment]
Jenkins test this please

File: python/tests/share/test_shell_jobs.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                fm_shell_jobs[job.name().lower()] = exe
+
+        list_from_content = res_config.ert_workflow_list
+        wf_shell_jobs = {}
+        for wf_name in list_from_content.getJobNames():
+            exe = list_from_content.getJob(wf_name).executable()
+            if exe and exe.startswith(share_path):
+                wf_shell_jobs[wf_name] = exe
+
+        assert fm_shell_jobs == wf_shell_jobs
```
Comment (mortalisk) [inline python/tests/share/test_shell_jobs.py:27]: On my machine, both these dicts end up as empty when I run this test. ( I assume they are supposed to contain something )

File: python/tests/share/test_shell_jobs.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from res.job_queue import ExtJob
```
Comment (mortalisk) [inline python/tests/share/test_shell_jobs.py]: Some unused imports

File: python/tests/share/test_shell_jobs.py (hunk 1 above)
Comment (jepebe) [reply python/tests/share/test_shell_jobs.py:27]: You need the `site-config` to include the workflows in the *share* directory. But I will add a test with names as well.

Comment (mortalisk) [reply python/tests/share/test_shell_jobs.py:27]: Can we make the site-config explicit in the test, so that it runs the same everywhere?

Comment (jepebe) [reply python/tests/share/test_shell_jobs.py:27]: Well it works on my machine...

File: python/tests/share/test_shell_jobs.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        list_from_content = res_config.ert_workflow_list
+        wf_shell_jobs = {}
+        for wf_name in list_from_content.getJobNames():
+            exe = list_from_content.getJob(wf_name).executable()
+            if exe and exe.startswith(share_path):
+                wf_shell_jobs[wf_name] = exe
+
+        assert fm_shell_jobs == wf_shell_jobs
+
+    def test_shell_script_jobs_names(self):
```
Comment (mortalisk) [inline python/tests/share/test_shell_jobs.py:29]: This test fails on my machine. How do I use the correct site-config?

Comment (jepebe) [reply python/tests/share/test_shell_jobs.py:29]: Working on it. :) "It works on my machine" as in failing on Jenkins and Travis

File: python/tests/share/test_shell_jobs.py (hunk 1 above)
Comment (jepebe) [reply python/tests/share/test_shell_jobs.py:27]: Maybe a bug but `createSharePath` and the actual executable location differ when running on Travis/Jenkins. The share path is located outside of the **venv** and the executable is inside it. I have worked around that issue in the updated test.

File: share/ert/site-config [hunk 4]
```diff
@@ -1,5 +1,6 @@
 -- This is a very basic site config file, mainly used for testing.
 
+WORKFLOW_JOB_DIRECTORY workflows/jobs/shell
```
Comment (sondreso) [inline share/ert/site-config:3]: The equivalent of this line also needs to be added here ([link](https://github.com/equinor/ert/blob/8f2e7181123c53555520b1d3359b15c893b3b5e3/ert_shared/hook_implementations/jobs.py#L103)) for the plugin system to install them 🙂

Comment (mortalisk) [reply share/ert/site-config:3]: Will you make an issue for doing that as well? After doing that this looks good to me.

**Review by mortalisk — APPROVED** [review]

File: share/ert/site-config (hunk 4 above)
Comment (jepebe) [reply share/ert/site-config:3]: Already made a PR Equinor/ert#1029.

**Pull request merged**